### ISSUE TITLE
Docker image doesn't build

### ISSUE THREAD

**Issue by davejlong** [kittoframework/kitto#8]
I've been having an issue getting the Docker image to build for my dashboard. Anytime I try to run the build it fails on the `contextify` dependency. Contextify seems to require gcc, python2 and make so I had to add the following to my Dockerfile:

```
RUN apt-get update && apt-get -yqq install build-essential python \
  && apt-get clean \
  && rm -rf /var/lib/apt/lists/*
```

Obviously this isn't super great because it adds a lot more to my image in terms of complexity and size.

**Comment by davejlong** [comment]
I didn't realize the issue until I added a `.dockerignore` to my project to ignore `node_modules`, `deps` and `_build` because previous to that contextify was installed fine from my host machine.

**Comment by zorbash** [comment]
Since [contextify](https://github.com/brianmcd/contextify) is an npm dependency I'm guessing that the image fails to build during npm install. Can you post the complete docker build output together with your `docker -v` output?

**Comment by zorbash** [comment]
I managed to reproduce the issue and have contextify installation failing. Seems related to https://github.com/brianmcd/contextify/issues/188.

The thing is that the package clearly states:

`YOU DON'T NEED THIS MODULE ON NODE >= 0.12`

and the Dockerfile installs node@4.3.1

I think we have to find a way not to depend on this package.

**Comment by zorbash** [comment]
Fixed by acbda8855
